turbo.json

```diff
@@ -2,12 +2,21 @@
   "$schema": "https://turbo.build/schema.json",
   "pipeline": {
     "build": {
-      "outputs": ["build/**", ".next/**", "!.next/cache/**"],
+      "outputs": [
+        "build/**",
+        ".next/**",
+        "!.next/cache/**",
+        "src/codegen/**",
+        "scripts/generated/**",
+        "config/wallet-registry.ts",
+        "config/wallet-assets.ts",
+        "config/chain-infos.ts"
```

Comment:
Why not put these in a generated folder in config folder? Would also help inform from import paths that it's using generated stuff?

IMO, generated stuff makes things super confusing for new devs.. let's isolate it as much as possible, as we do with integrations folder.

Bonus: we can refer to it in a single blob if everything generated is in a generated folder (including codegen): 

```
["src/generated/**", "src/**/generated/**"]
```